server/src/infra/ldap_handler.rs

```diff
@@ -204,6 +210,333 @@ fn root_dse_response(base_dn: &str) -> LdapOp {
                 atype: "isGlobalCatalogReady".to_string(),
                 vals: vec![b"false".to_vec()],
             },
+            LdapPartialAttribute {
+                atype: "subschemaSubentry".to_string(),
+                vals: vec![b"cn=Subschema".to_vec()],
+            },
+        ],
+    })
+}
+
+pub struct ObjectClassList(Vec<LdapObjectClass>);
+
+impl ObjectClassList {
+    fn format_for_schema(&self) -> String {
+        self.0
+            .iter()
+            .map(|c| format!("'{}'", c))
+            .unique()
+            .collect::<Vec<_>>()
+            .join(" ")
+    }
+}
+
+pub struct LldapBuiltinSchema {
```

Comment:
I'm not a big fan of this name. Lldap doesn't bring much value :)
And built-in sounds like hardcoded. Maybe LdapSchemaDescription?

And let's add a link to the RFC section, as well as a comment for what this struct represents.